src/main/java/zowe/client/sdk/zosfiles/uss/methods/UssWrite.java

```diff
@@ -9,5 +9,97 @@
  */
 package zowe.client.sdk.zosfiles.uss.methods;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import zowe.client.sdk.core.ZosConnection;
+import zowe.client.sdk.rest.*;
+import zowe.client.sdk.rest.type.ZoweRequestType;
+import zowe.client.sdk.utility.RestUtils;
+import zowe.client.sdk.utility.ValidateUtils;
+import zowe.client.sdk.zosfiles.ZosFilesConstants;
+
+/**
+ * Provides unix system service write to object functionality
+ * <p>
+ * <a href="https://www.ibm.com/docs/en/zos/2.4.0?topic=interface-write-data-zos-unix-file">z/OSMF REST API</a>
+ * 
+ * @author James Kostrewski
+ * @version 2.0
+ */
 public class UssWrite {
+
+    private static final Logger LOG = LoggerFactory.getLogger(UssDelete.class);
+    private final ZosConnection connection;
+    private ZoweRequest request;
+
+    /**
+     * UssWrite Constructor
+     *
+     * @param connection connection information, see ZOSConnection object
+     * @author James Kostrewski
+     */
+    public UssWrite(ZosConnection connection) {
+        ValidateUtils.checkConnection(connection);
+        this.connection = connection;
+    }
+
+    /**
+     * Alternative UssWrite constructor with ZoweRequest object. This is mainly used for internal code
+     * unit testing with mockito, and it is not recommended to be used by the larger community.
+     *
+     * @param connection connection information, see ZOSConnection object
+     * @param request    any compatible ZoweRequest Interface type object
+     * @author James Kostrewski
+     * @author Frank Giordano
+     */
+    public UssWrite(ZosConnection connection, ZoweRequest request) {
+        ValidateUtils.checkConnection(connection);
+        this.connection = connection;
+        this.request = request;
+    }
+    
+    /**
+     * Replaces the content of a unix file with new content.
+     *
+     * @param destName destination name of where the file is located
+     * @param memberName  name of member to add new content
+     * @param content     new content
+     * @return http response object
+     * @throws Exception error processing request
+     * @author Frank Giordano
+     */
+    public Response write(String destName, String memberName, String content) throws Exception {
```

Comment:
this write method wont work for updating/writing to file. Correct? It seems to be a copy and paste from DSN package. We can remove it.

Reply:
yes guilty, i think i requested the pull too early

Reply:
no problem. clean it up.. you can continue to checkin while PR is open and it gets updated.